clients/extension/src/utils/tx/getStandardTx.ts

```
@@ -71,9 +72,9 @@
       : undefined,
     data: tx.data,
     gasSettings: {
-      maxFeePerGas: tx.maxFeePerGas,
-      maxPriorityFeePerGas: tx.maxPriorityFeePerGas,
-      gasLimit: tx.gas,
+      maxFeePerGas: ethers.isHexString(tx.maxFeePerGas)?ethers.toBigInt(tx.maxFeePerGas).toString(): tx.maxFeePerGas,
```

ESLint

[error] Replace `?ethers.toBigInt(tx.maxFeePerGas).toString()` with `⏎········?·ethers.toBigInt(tx.maxFeePerGas).toString()⏎········`

(prettier/prettier)